hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -285,7 +285,7 @@ def exchange_create():
     api_type_name = data.get("api")
 
     if not re.match("^[A-Z0-9_]+$", bank):
-        abort(400, "Field `bank` must match /^[A-Z0-9_]$/")
+        abort(400, "Field `bank` must match /^[A-Z0-9_]+$/")
```

Comment:
I think this was a typo? Updated it because the message doesn't match regex actually being called?

Reply:
Yup, typo. We can also explain it in english if that's easier.